### PR TITLE
DLS-7101 NINO navigation

### PR DESCRIPTION
**Pull request by jackwillett1**

### CODE REVIEW

File: it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnerNinoControllerIntegrationSpec.scala
Comment (gjstirling) [inline it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnerNinoControllerIntegrationSpec.scala:1]: Missing IT test for pressing back button and navigation when user clicks "save and come back later" link.

**Review by gjstirling — COMMENTED** [review]
Scenario 2 and 5 have not been met. Back button has no link.

**Comment by Chrissystk** [comment]
> Scenario 2 and 5 have not been met. Back button has no link.

The back button does have a link it just cannot be tested right now as you get to the page by directly typing the url rather than going from the previous page. Once the navigation from the previous page is complete this link will work.

**Review by gjstirling — APPROVED** [review]
Feedback from Christine, good to go.

**Pull request merged**